src/libs/Navigation/Navigation.ts

```diff
@@ -158,6 +158,14 @@ function getActiveRoute(): string {
     return '';
 }
 
+/**
+ * Checks whether the report is opened in RHP (from search results)
+ */
+function getIsReportOpenInRHP(): boolean {
+    const rootState = navigationRef.getRootState();
```

Comment:
NAB just wanted to drop an opinion that it should also be legit to get navigation ref via this:
`import navigationRef from '@navigation/navigationRef';`

and with this you could just import the ref directly in HeaderView and moneyRequestHeader etc. and this function from `Navigation` could be removed in favor of directly using `isReportOpenInRHP`.

I don't string feelings about this, but I think if the only thing this function does is to grab `rootState` then it's not that useful

Reply:
I agree with @Kicu, let's handle it directly in files where it has to be used. Now we're working with @adamgrzybowski on refactoring the navigation structure and trying to reduce the number of utility functions in `Navigation.ts`.